### ISSUE TITLE
How searching for diatomite column in postgres works? 

### ISSUE THREAD

**Issue by dpsk** [jbox-web/ajax-datatables-rails#132]
I'm don't see it being casted in the sql query, should i throw some custom code and override searching methods?
Using 0.3.1

**Comment by ajahongir** [comment]
@dpsk можно фильтровать так:

``` ruby
def get_raw_records
   User.where(name: params[:name])
end
```

**Comment by dpsk** [comment]
@ajahongir so it doesn't let you to filter datetime by simply adding them into searchable_columns array?

**Comment by ajahongir** [comment]
@dpsk no, its not implemented on 0.3 you have to look at 0.4 https://github.com/ajahongir/ajax-datatables-rails-v-0-4-0-how-to 

here you can define columns as:

``` ruby
def view_columns
    @view_columns ||= {
      id: { source: "City.id", cond: :eq },
      created_at: { source: "City.created_at", cond: :gt },
      ....
  end
```
